src/status_im/group_chats/core.cljs

```diff
@@ -115,7 +115,11 @@
                                 :dsts          members
                                 :success-event [:transport/message-sent
                                                 chat-id
-                                                (transport.utils/message-id (:message payload))
+                                                (transport.utils/message-id
+                                                 ;; NOTE: There is no clock-value here.
+                                                 ;; Will we have collision?
+                                                 {:from current-public-key
```

Comment:
there will definitely be collisions with these params what would be the consequences?

Reply:
Idk, it seems that we are not storing those messages in DB or i am wrong?